kb/how-to/rewrite-branch-for-coding-style.md

```diff
@@ -40,7 +40,7 @@ If there is an ongoing review of a pull request, please communicate with the rev
 Please have the following tools and files available:
 
 * Git 2.17.0 or newer.
-* [Uncrustify](https://github.com/uncrustify/uncrustify) 0.75.1. Please note that older or newer versions are likely not to give the desired result.
+* [Uncrustify](https://github.com/uncrustify/uncrustify) 0.75.1. Please note that older or newer versions are likely not to give the desired result. See the [“Installing uncrustify”](uncrustify.md#installing-uncrustify) article for some tips.
```

Comment:
Should we use the same text as in `mbedtls-coding-standards.md`? 

`older or newer versions are not suitable because they give different results`